types/recharts/recharts-tests.tsx

```diff
@@ -15,12 +15,12 @@ const Component = (props: {}) => {
     ];
 
     return (
-        <LineChart width={500} height={300} data={ data }>
+        <LineChart width={500} height={300} data={data}>
             <XAxis dataKey="name"/>
-            <YAxis />
-            <CartesianGrid />
-            <Line type="monotone" dataKey="uv" points={ [] } />
-            <Line type="monotone" dataKey="pv" points={ [] } />
+            <YAxis/>
+            <CartesianGrid stroke="#eee" strokeDasharray="5 5"/>
```

Comment:
I copied the example from the starting page of recharts, and made it work with the extension from CSSStyleDeclaration of the component which can be styled.